Common/Securities/SecurityTransactionModel.cs

```diff
@@ -323,15 +313,14 @@ public virtual decimal GetOrderFee(decimal quantity, decimal price)
         /// <returns>The cost of the order in units of the account currency</returns>
         public virtual decimal GetOrderFee(Security security, Order order)
         {
-            if (order.Quantity == 0)
+			if (order.Quantity == Decimal.Zero)
             {
-                return 0m;
+				return Decimal.Zero;
```

Comment:
I think these changes are just style and can cause churn in the code base. Also, please keep code changes/white space changes separate.